framework/src/main/java/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -13,6 +13,8 @@
 import org.checkerframework.dataflow.analysis.RegularTransferResult;
 import org.checkerframework.dataflow.analysis.TransferInput;
 import org.checkerframework.dataflow.analysis.TransferResult;
+// TODO: remove this line after merging StringConcatenateAssignmentNode completely.
+import org.checkerframework.dataflow.cfg.node.*;
```

Comment:
Unfortunately, a format check fails because of this.
Is there no cleaner way to suppress the deprecation warning?

Reply:
fixed.

Reply:
fixed.